searx/webapp.py

```diff
@@ -382,6 +383,11 @@ def url_for_theme(endpoint: str, override_theme: Optional[str] = None, **values)
         if file_hash:
             values['filename'] = filename_with_theme
             suffix = "?" + file_hash
+    if endpoint == 'info' and 'locale' not in values:
```

Comment:
see `simple/base.html`  and `oscar/nav.html`.

allow to have a working link to the about page even in a 404 page.